src/static/js/modules/util/breakpoint-state.js

```diff
@@ -0,0 +1,39 @@
+/* ==========================================================================
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var _breakpointsConfig = require( '../../config/breakpoints-config' );
+var _getViewportDimensions = require( './get-viewport-dimensions' )
+                             .getViewportDimensions;
+
+function _inBreakpointRange( breakpointRange, width ) {
+  var min = breakpointRange.min || 0;
+  var max = breakpointRange.max || Number.POSITIVE_INFINITY;
+
+  return min <= width && width <= max;
+}
+
+/**
+ * @returns {object} An object literal with Boolean
+ * isMobile, isTablet, isDesktop, isWall, isSmall properties.
+ * @param {integer} width Current window width.
+ */
+function get( width ) {
+  var breakpointState = {};
+  var breakpointKey;
+  width = width || _getViewportDimensions().width;
+
+  for ( var rangeKey in _breakpointsConfig ) { // eslint-disable-line guard-for-in, no-inline-comments, max-len
```

Comment:
I think the idea is since it goes through code that we don't control (Browserify in this case) it safeguards this code in case the prototype were to be augmented at some point. But you're right, it's not very likely. And eventually we may want to replace these with Object.keys() anyway.